frontend/src/components/TaskTable.tsx

```diff
@@ -69,6 +82,67 @@ export const TaskTable: React.FC<{
     return sorted;
   };
 
+  // Return tasks that don't have ratings from everyone involved in the task - View for product owner
+  const getOwnerTask: () => Task[] = () => {
+    const developers = allUsers?.filter(
+      (user) => user.type === RoleType.Developer,
+    );
+    const unrated = getRenderTaskList().filter((task) => {
+      const ratingIds = task.ratings.map((rating) => rating.createdByUser);
+
+      const unratedCustomers = allCustomers?.filter((customer) => {
+        const representativeIds = customer?.representatives?.map(
+          (rep) => rep.id,
+        );
+        return !representativeIds?.every((id) => ratingIds?.includes(id));
+      });
+
+      const missingDevs = developers?.filter(
+        (developer) => !ratingIds.includes(developer.id),
+      );
+
+      if (
+        unratedCustomers &&
+        unratedCustomers?.length < 1 &&
+        missingDevs &&
+        missingDevs?.length < 1
+      )
+        return false;
+      return true;
+    });
+
+    return unrated;
+  };
+
+  // Return tasks that are not rated by logged in user
+  const getUnratedTasks: () => Task[] = () => {
+    return getRenderTaskList().filter(
+      (task) =>
+        !task.ratings.find((rating) => rating.createdByUser === userInfo!.id),
+    );
```

Comment:
Tällä hetkellä Business-RoleTypella taskit menee waiting for ratings-kohtaan, jos ei edusta ketään
![image](https://user-images.githubusercontent.com/80758782/123599392-370a0280-d7fe-11eb-85fd-146cf46c5238.png)
utils/taskUtils.ts taskAwaitsRatings-funktiota voisi käyttää tässä hyödyksi: se tsekkaa puuttuuko taskista käyttäjän arvio ottaen roolit huomioon